apps/heureka/src/components/filters/FilterSelect.jsx

```diff
@@ -85,7 +85,7 @@ const FilterSelect = ({ entityName, isLoading }) => {
       {activeFilters && Object.keys(activeFilters).length > 0 && (
         <Button label="Clear all" onClick={() => clearActiveFilters(entityName)} variant="subdued" />
       )}
-      {entityName === "IssueMatches" && (
+      {["IssueMatches", "Services"].includes(entityName) && (
```

Comment:
wouldn't be better to pass to the component a list of options which enables or disables specific elements instead of matching entity names to know if you should render something?